mriqc/interfaces/bids.py

```diff
@@ -85,6 +90,11 @@ def __setattr__(self, key, value):
                 self._outputs[key] = value
             super(IQMFileSinkInputSpec, self).__setattr__(key, value)
 
+        if key=='subject_id':
```

Comment:
As said this is a dirty fix, as I could not find where to set the subject id from outside ...
Ideally I would like to have the subject_id given in the command line, and set but where ...
any idea is wellcome